src/components/PCAPTable.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
```

Comment:
Also since this is our first websocket project, let's be liberal in documenting. The component name can be later changed, but let's see what it can be.

Reply:
Okay